EIPS/eip-5792.md

```diff
@@ -207,23 +169,66 @@ As with the return value of `wallet_sendCalls`, the batch identifier may be any
 }
 ```
 
-### `atomicBatch` Capability
+### `wallet_getCapabilities`
+
+An app calling the above-defined methods without prior knowledge that the wallet supports them risks degrading user experience considerably, so an additional RPC for querying the wallet's capabilities is provided.
```

Comment:
Capabilities is not for discovering which RPCs that wallet supports.

Reply:
Or maybe you mean, which capabilities the wallet supports?

Reply:
very good feedback.  resolving because of [this commit](https://github.com/ethereum/EIPs/pull/8387/commits/4e7cb461c846050681a322a4bb738ed29c969034) but feel free to comment again if the new prose is just as off-base in another direction!